core/psh/sysexec.c

```diff
@@ -25,6 +27,63 @@ void psh_sysexecinfo(void)
 }
 
 
+static int psh_sysexec_wldargmatch(char *wld, int argc, char **argv, char wldend, int argmaxlen)
+{
+	int len = 0, ii;
+	char *currc = wld, *nextc, *cmdend;
+
+	cmdend = strchr(wld, wldend);
+	currc = wld;
```

Comment:
This assignment is redundant, because You've already assigned `currc = wld`, 3 lines above, since then pointer `wld` has not changed.

Reply:
Yes, you are right. Redundancy removed.